### PR TITLE
Add --puppet-noop switch. Update README

### LINKED ISSUE #173: [RFE] Allow user to set 'noop=true' in the puppet.conf file.
From [RHBZ1435972](https://bugzilla.redhat.com/show_bug.cgi?id=1435972) - As a user, I'd like the ability to set `noop = true` in the puppet.conf. This allows me to gain insight into how a migrated or registered host differs from my desired puppet state without making changes on the client. 

It is assumed that the admin will manually remove the `noop = true` directive later.

### PR DESCRIPTION
**Pull request by sideangleside**
Fixes  #173

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -338,13 +338,28 @@ On machines with multiple interfaces or multiple addresses on one interface, it
...
+
+
+
 # Help / Available options:
 
 ~~~
 Foreman Bootstrap Script
 This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello
 Usage: bootstrap.py [options]
-
```
Comment (evgeni) [inline README.md]: wat? :)

File: README.md [hunk 2]
```diff
@@ -393,7 +408,8 @@ Options:
   -f, --force           Force registration (will erase old katello and puppet
                         certs)
   --add-domain          Automatically add the clients domain to Foreman
-  --remove              Instead of registering the machine to Foreman remove it
+  --puppet-noop         Configure Puppet agent to only run in noop mode
+  --remove              Instead of registring the machine to Foreman remove it
```
Comment (evgeni) [inline README.md]: according to the internet™ it's registering: https://en.wiktionary.org/wiki/registering

**Review by evgeni — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (sideangleside) [reply README.md]: The script itself had the typo in its `--help` output, and I inadvertently copy/pasted it.

**Comment by sideangleside** [comment]
Fixed up the two typos in the README.md and bootstrap.py's `-h` output & squashed the two commits together.

**Pull request merged**